wrapanapi/systems/ec2.py

```diff
@@ -784,6 +784,9 @@ def create_s3_bucket(self, bucket_name):
             self.logger.exception("Error: Bucket was not successfully created.")
             return False
 
+    def list_s3_bucket(self):
+        return [bucket for bucket in self.s3_connection.buckets.all()]
```

Comment:
Required:
I think better to go with `bucket.name`. what do you think?

Reply:
Thanks @digitronik , I will update it.